### PR TITLE
Update dependencies to the latest timestamp version

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -13,8 +13,8 @@ ballerinaGradlePluginVersion=0.14.1
 ballerinaLangVersion=2201.0.1
 
 #stdlib dependencies
-stdlibIoVersion=1.2.1
-stdlibRegexVersion=1.2.1
+stdlibIoVersion=1.2.2-20220407-142300-7060eb4
+stdlibRegexVersion=1.3.0-20220425-094000-d073680
```
Comment (daneshk) [inline gradle.properties:17]: As discussed yesterday, when we are moving to minor versions of the dependent modules. we need to move our modules to minor versions.

File: ballerina/Dependencies.toml [hunk 2]
```diff
@@ -55,7 +55,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "observe"
-version = "1.0.2"
+version = "1.0.4"
```
Comment (niveathika) [inline ballerina/Dependencies.toml:58]: Minor version updates of dependencies are not reflected here, If there are changes to minot version. Delete the dependencies toml and run the build

File: gradle.properties (hunk 1 above)
Comment (MadhukaHarith92) [reply gradle.properties:17]: Updated module version to `2.3.0`

File: ballerina/Dependencies.toml (hunk 2 above)
Comment (MadhukaHarith92) [reply ballerina/Dependencies.toml:58]: The minor version dependency update is related to the `integration-tests` package `Dependencies.toml` file which is not being committed.

**Review by niveathika — APPROVED** [review]

**Pull request merged**